app/views/spell_lists/_form.haml

```diff
@@ -0,0 +1,13 @@
+= simple_form_for @spell_list do |f|
+  = f.input :title
+  = f.association :spells, as: :check_boxes, label: false, label_method: :multiple_labels, collection: Spell.order('level, name').where('level = 0')
```

Comment:
Two notes here:

1. Probably could use a named scope methods in `Spell` model, you should never write sql in views. Maybe something like `for_level` or `with_level`? same for the `order` scope.
2. Probably better would be to just group spells in sql by level and display association this way, since now for 7 levels you are firing up 7 different queries from view to DB.

Reply:
I mostly get it, and scopes got implemented - but I still am a bit perplexed how to do this really DRY...? Later on I plan to create separators in a form of level titles ("Level 1", "Level 2" and so on) - so that gets tricky.